src/customFilters.js

```diff
@@ -0,0 +1,85 @@
+const Markdown = require('markdown-it');
+const _ = require('lodash');
+const OpenAPISampler = require('openapi-sampler');
+
+const filter = module.exports;
+
+/**
+Filters that go away by adding loadash:
```

Comment:
```suggestion
Filters that go away by adding lodash:
```

Reply:
ups, this is actually something that I should remove from PR as it was just a note for me :)